mlrun/api/api/endpoints/grafana_proxy.py

```diff
@@ -0,0 +1,220 @@
+from http import HTTPStatus
+from typing import List, Dict, Any, Callable
+
+from fastapi import APIRouter, Response, Request
+
+from mlrun.api.api.endpoints.model_endpoints import get_access_key, list_endpoints
+from mlrun.api.schemas import ModelEndpointStateList, GrafanaTable, GrafanaColumn
+from mlrun.config import config
+from mlrun.errors import MLRunBadRequestError
+from mlrun.utils.v3io_clients import get_frames_client
+
+router = APIRouter()
+
+
+@router.get("/grafana-proxy/model-endpoints", status_code=HTTPStatus.OK.value)
+def grafana_proxy_model_endpoints_check_connection(request: Request):
+    """
+    Root of grafana proxy for the model-endpoints API, used for validating the model-endpoints data source
+    connectivity.
+    """
+    get_access_key(request)
+    return Response(status_code=HTTPStatus.OK.value)
+
+
+@router.post("/grafana-proxy/model-endpoints/query", response_model=List[GrafanaTable])
```

Comment:
And if the above can be `/grafana-proxy/health` I guess this one can be `/grafana-proxy/model-endpoints`

Reply:
Same as above